src/createWebpackLessPlugin.js

```diff
@@ -18,7 +18,7 @@ const matchMalformedModuleFilename = /(~[^/\\]+)\.less$/;
 // This somewhat changed in Less 3.x. Now the file name comes without the
 // automatically added extension whereas the extension is passed in as `options.ext`.
 // So, if the file name matches this regexp, we simply ignore the proposed extension.
-const isModuleName = /^~[^/\\]+$/;
+const isModuleName = /^~(|@[a-z0-9-~][a-z0-9_.-~]*\/)[^/\\]+$/;
```

Comment:
I think we really no need this complex regexp, simplify this as soon as possible, if package name was invalid we anyway throw error, so no need complex check

Reply:
updated with `/^~(@[^/]+\/)?[^/\\]+$/`